app/javascript/components/project-submissions/components/modal.jsx

```diff
@@ -2,19 +2,31 @@ import React from 'react';
 import { func, bool, node } from 'prop-types';
 import ScrollLock from 'react-scrolllock';
 
+const modalShown = 'fixed inset-0 bg-black bg-opacity-60 z-50';
+const modalBodyPosition = 'fixed top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2';
+const modalCloseBtn = `
+  absolute inline
+  top-4 right-4 m-0 p-0 border-0
+  bg-transparent
+  text-light-accent hover:text-black text-xl font-black font-close-button
+  cursor-pointer
+`;
+
 const Modal = ({ handleClose, show, children }) => {
-  const showHideClassName = show ? 'react-modal' : 'react-modal--hidden';
+  const showHideClassName = show ? modalShown : 'hidden';
 
   return (
     <ScrollLock isActive={show}>
       <div className={showHideClassName} aria-label={show ? 'modal--shown' : 'modal--hidden'}>
-        <div className="react-modal__body">
+        <div className={`${modalBodyPosition} h-auto p-10 rounded-xl bg-white w-11/12 sm:w-4/5 md:w-3/5 lg:w-2/5`}>
           <button
-            className="link-button react-modal__close-btn"
+            className={modalCloseBtn}
             onClick={handleClose}
             type="button"
             aria-label="close"
-          />
+          >
+            x
```

Comment:
I've been testing a little in the review app. The modal close button doesn't look like it's rendering the font awesome version of the x unfortunately.

_I think_ we can use an icon element here for the font awesome icon instead 
`<i class="fa-solid fa-x"></i>`

Reply:
Figured it out, finally!